me.stonegate.tsacdop.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Name=Tsacdop
+Icon=me.stonegate.tsacdop
+Exec=env GDK_GL=gles tsacdop %U
```

Comment:
you should set the env in finish args instead. Also `gles` is not on x86_64 which is the only platform you seem to support, so I don't see why this would be necessary.

Reply:
Thanks I will remove it.